src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.html

```diff
@@ -1,6 +1,9 @@
 <div *ngIf="exercise">
     <h4 jhiTranslate="artemisApp.fileUploadAssessment.dashboard.heading" [translateValues]="{ exerciseTitle: exercise.title }"></h4>
-    <jhi-assessment-filters [submissions]="submissions" (filterChange)="updateFilteredSubmissions($event)"></jhi-assessment-filters>
+    <jhi-assessment-filters [submissions]="submissions" (filterChange)="applyChartFilter($event)" [resetFilter]="resetFilter"></jhi-assessment-filters>
+    <div *ngIf="filterOption !== undefined" type="button" class="btn btn-primary" (click)="resetFilterOptions()" jhiTranslate="artemisApp.assessment.dashboard.resetFilter">
```

Comment:
```suggestion
    <div *ngIf="!!filterOption" type="button" class="btn btn-primary" (click)="resetFilterOptions()" jhiTranslate="artemisApp.assessment.dashboard.resetFilter">
```

Reply:
I might be wrong on this, but I think this would also hide the button if `filterOption === 0`. Since this is a possible filter value, we need the reset button in this case as well. :)

Reply:
Than we can leave it as it is :+1: